pallets/subbridge/src/xcmbridge.rs

```diff
@@ -24,7 +24,6 @@ pub mod pallet {
 	#[pallet::pallet]
 	#[pallet::generate_store(pub(super) trait Store)]
 	#[pallet::storage_version(STORAGE_VERSION)]
-	// TODO: remove when we Vec get replaced by BoundedVec
 	#[pallet::without_storage_info]
```

Comment:
Now we can remove `without_storage_info`

Reply:
Removed